assets/src/edit-story/components/panels/design/textBox/padding.js

```diff
@@ -138,29 +151,34 @@ function PaddingControls({
         stretch: true,
       }
     : {
-        suffix: _x('H', 'The Horizontal padding', 'web-stories'),
+        suffix: __('Horizontal padding', 'web-stories'),
         'aria-label': __('Horizontal padding', 'web-stories'),
-        onChange: (value) =>
+        onChange: (evt) => {
+          const value = Number(evt?.target?.value);
           handleChange((el) => ({
             horizontal: value + getHiddenPadding(el, 'horizontal'),
-          })),
+          }));
+        },
       };
 
   return (
-    <Row>
-      <Label>{__('Padding', 'web-stories')}</Label>
-      <BoxedNumeric
-        value={displayedPadding.horizontal}
-        {...firstInputProperties}
-      />
-      <Space />
-      <Toggle
-        icon={<Lock />}
-        uncheckedIcon={<Unlock />}
-        value={lockPadding}
-        onChange={() =>
-          handleChange(
-            (el) =>
+    <>
+      <Row>
+        <NumericInput
+          value={displayedPadding.horizontal}
+          isIndeterminate={displayedPadding.horizontal === MULTIPLE_VALUE}
```

Comment:
This is not really working currently, doesn't allow changing multi-value at all.

Reply:
Hm, it looks like we need to rework the numeric input a bit for the `inDeterminate` logic to work for the multiple value, it has an internal change handler that interferes with it. cc @samwhale

Reply:
Oh okay! I tried to make this component as composable as possible, sorry if it's being annoying.

What do we need the `isIndeterminate` logic for?  From what I'm seeing is it to make sure that the validation runs and we know that whatever value is shown is 'valid'?

Reply:
It seems like @samwhale 's solution here: #6604 fixes it for multiple value?

Reply:
Let's use consistent logic across the panels, currently, `isIndeterminate` is used elsewhere, too. #6640 added to follow up.